wiki/Play_style/Alternating/en.md

```diff
@@ -17,10 +17,10 @@ tags:
 
 Alternating has a wide variety of uses and its forms can vary from player to player. For example, many players who alternate typically switch between `Z` and `X` on every other note (or hit object), while others switch keys after every two to three hit objects. It should be noted that some players will alternate most of the time, but sometimes hit the same key twice if a hit object catches them off-guard. This is all simply down to personal preference.
 
-On a similar note, some players prefer to tap their keys using the index and middle finger, while others use their index and ring finger. Index and ring finger players will typically have their keys set farther apart to accommodate for the larger gap. E.g., `Z` and `C` as apposed to `Z` and `X`.
+On a similar note, some players prefer to tap their keys using the index and middle finger, while others use their index and ring finger. Index and ring finger players will typically have their keys set farther apart to accommodate for the larger gap. E.g., `Z` and `C` as opposed to `Z` and `X`.
 
 ## Arguments
 
-Arguments in-favor of alternating typically mention that alternating is much faster as it reduces the amount of time between keypresses as apposed to [single-tapping](/wiki/Play_style/Single-tapping) and that it gives time for each finger to rest and thus allows players to play faster for longer.
+Arguments in-favour of alternating typically mention that alternating is much faster as it reduces the amount of time between keypresses as opposed to [single-tapping](/wiki/Play_style/Single-tapping) and that it gives time for each finger to rest and thus allows players to play faster for longer.
```

Comment:
to my knowledge, this is how it should be written. also happens in other Play style articles, as they are likely written by the same person
```suggestion
Arguments in favour of alternating typically mention that alternating is much faster as it reduces the amount of time between keypresses as opposed to [single-tapping](/wiki/Play_style/Single-tapping) and that it gives time for each finger to rest and thus allows players to play faster for longer.
```